sdcm/cluster_k8s/gke.py

```diff
@@ -505,7 +505,8 @@ def add_nodes(self,
                   ec2_user_data: str = "",
                   dc_idx: int = None,
                   rack: int = 0,
-                  enable_auto_bootstrap: bool = False) -> List[GkeScyllaPodContainer]:
+                  enable_auto_bootstrap: bool = False,
+                  instance_type=None) -> List[GkeScyllaPodContainer]:
```

Comment:
In the EKS module the new param gets provided to the following call of the `super()` method.
So, need either to remove there or add it here the same way.

Reply:
forward the call to the super call, and there would be an assert there